website/docs/docs/dbt-versions/release-notes/73-Jan-2024/jan-sl-updates.md

```diff
@@ -0,0 +1,35 @@
+- **Simplified dimension resolution** &mdash; Instead of specifying the fully qualified dimension name (for example, `order__user__country`) in the group by or filter expression, you now only need to provide the primary entity and dimensions name, like `user__county`.
+- **Query saved queries** &mdash; You can now query the [saved queries](/docs/build/saved-queries) you've defined in the dbt Semantic Layer using [Tableau](/docs/use-dbt-semantic-layer/tableau), [GraphQL API](/docs/dbt-cloud-apis/sl-graphql), [JDBC API](docs/dbt-cloud-apis/sl-jdbc), and the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation).
+
+## Updates
+
+- **Display `label` for dbt Semantic Layer metrics** &mdash; The YAML spec parameter `label` is now available for Semantic Layer metrics in [JDBC and GraphQL APIs](/docs/dbt-cloud-apis/sl-api-overview). This means you can conveniently use `label` as a display name for your metrics when exposing them.
+- **Use shorthand to create metrics** &mdash; Added support for `create_metric=true` for a measure, which is a shorthand to quickly create metrics. This is especially useful in cases when metrics are only used to build other metrics.
+- **Support for Tableau Parameter filters** &mdash; Added support for Tableau Parameter filters. You can use [our Tableau connector](docs/use-dbt-semantic-layer/tableau) to create and use parameters with your dbt Semantic Layer data.
+- **Additional parameters for GraphQL API** &mdash;  Added support to expose `expr` and `agg` for [Measures](/docs/build/measures) in the [GraphQL API](/docs/dbt-cloud-apis/sl-graphql).
+- **Improved error messages** &mdash; When querying a dimension that is not reachable for a given metric, you now have improved error messages returned in the command line interface.
+- **Entities in Tableau** &mdash; We now surface entities in our Tableau integration and these can be queried like dimensions.
+- **New Tableau Data Source** &mdash; We introduced a new data source in our Tableau integration called "ALL", which contains all semantic objects defined. This has the same information as "METRICS_AND_DIMENSIONS". In the future, we will deprecate "METRICS_AND_DIMENSIONS" in favor of "ALL" for clarity. 
+
+## Bug fixes
+
+- **BigQuery numeric types** &mdash; Numeric types with precision greater than 38 (like `BIGDECIMAL`) in BigQuery are now supported and accepted. Previously, it was unsupported and would return an error.
+- **Support for scientific notation** &mdash; Scientific notation for large numbers is displayed and correctly interpreted. Previously, they were unsupported. 
+- **Google Sheets dimension values** &mdash; Dimension values are now accurately preserved instead of being inadvertently converted into strings. Previously, dimension values were incorrectly converted into a string.  
```

Comment:
sugg: repeating info

```suggestion
- **Google Sheets dimension values** &mdash; We now preserve dimension values accurately instead of being inadvertently converted into strings. 
```